addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -3467,15 +3475,30 @@ export class Wysiwyg extends Component {
      * @param {Element} el
      * @param {string} resModel
      * @param {number} resId
+     * @param {string} resField
+     * @param {string} type
      */
-    async _saveModifiedImage(el, resModel, resId) {
+    async _saveModifiedImage(el, resModel, resId, resField, type) {
+        const camelToSnakeCase = (dictionary) => {
+            // Convert all the keys of a dictionary from camel to snake case
+            const snakeEntryDic = {};
+            const camelEntries = Object.keys(dictionary);
+            camelEntries.forEach((camelEntry) => {
+                const snakeEntry = camelEntry.replace(/([a-z])([A-Z])/g, "$1_$2").toLowerCase();
+                snakeEntryDic[snakeEntry] = dictionary[camelEntry];
+            });
+            return snakeEntryDic;
+        };
```

Comment:
Same remarks as for the opposite conversion - and as SOBO said, maybe to be extracted to a reusable place.